api/db/migrations/20250917000000_add-duration-column-to-complementary-certifications.js

```diff
@@ -0,0 +1,24 @@
+const TABLE_NAME = 'complementary-certifications';
+const COLUMN_NAME = 'duration';
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+const up = async function (knex) {
+  await knex.schema.table(TABLE_NAME, function (table) {
+    table.integer(COLUMN_NAME).nullable();
```

Comment:
A la demande de DATA, garder en tete de documenter toutes les nouvelles colonnes avec une description fonctionnelle de ce que represente la valeur, interessant pour ce genre de colonne "duration" qui ne comporte aucune indication autoportante sur le format (heures, minutes, secondes.. ?) ou sa signification fonctinnelle

```suggestion
    table.integer(COLUMN_NAME).nullable().comment('Represents the duration in minutes necessary to take the certification test');
```